### DISCUSSION TITLE
Constant interpolation type for gradients

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6489] [Ideas]
I think that the addition of a `Constant` interpolation type to gradients, like what is available in Blender, would be useful.

You can see here how this operates in Blender vs the typical `Linear` interpolation type:
![Blender Constant Interpolation](https://user-images.githubusercontent.com/42905003/80143682-1a908b80-85a5-11ea-8bd8-ea5b89a2f536.gif)

Creating perfectly sharp color bands in gradients is a bit finicky in OT as colors become difficult to select when placed close together as can be seen here, not sure if this should be counted as a bug:
![Gradient selection issue](https://user-images.githubusercontent.com/42905003/80144091-cd60e980-85a5-11ea-9468-94eb17853eb4.gif)
You can see that I am trying to select the white color, but the black color is instead selected when both colors are too close.
A `Constant` interpolation type would greatly reduce the amount of times this issue is encountered.

**Comment by gab3d** [comment]
agreed...
